cg/cli/workflow/taxprofiler/base.py

```diff
@@ -209,3 +210,6 @@ def start_available(context: click.Context, dry_run: bool = False) -> None:
             exit_code = EXIT_FAIL
     if exit_code:
         raise click.Abort
+
+
+taxprofiler.add_command(metrics_deliver)
```

Comment:
Could you move this under `taxprofiler.add_command(resolve_compression)` so we have them grouped?